nicolas-martin-castellanos/tarea-1/magazine1/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="box">
+      <div class="jordan">
```

Comment:
@NicolasMartin23 , remember you should be using semantic tags and not divs

Reply:
I made the correction and forwarded to PR.